### PR TITLE
YA

### PR DESCRIPTION
**Pull request by Yuhans**

### CODE REVIEW

File: src/main/java/com/epam/javacourses2016/task15/SolverTask15.java [hunk 1]
```diff
@@ -17,8 +19,30 @@
...
      */
     IFileWithLines analyze(Set<Point2D> points, File output) {
-        //TODO
-        return null;
+        Set<Line> lines = new HashSet<>();
+        ArrayList<Point2D> allPoints = new ArrayList<>(points);
+        int checkedPoints = 0;
+        for (Point2D point : points) {
+            checkedPoints++;
+            for (int i = checkedPoints; i < points.size(); i++) {
```
Comment (nkarulin) [inline src/main/java/com/epam/javacourses2016/task15/SolverTask15.java:27]: Bruteforce :fist: :snail:

**Comment by nkarulin** [comment]
Please enable tests 16 and 18.
Waiting for task 17, right?

**Comment by Yuhans** [comment]
Tasks 17, 18 haven't completed yet.

**Comment by Yuhans** [comment]
Oh, i missed, task17 is ready and test17 is already enabled

**Comment by nkarulin** [comment]
Ok. nice! Waiting for the last tasks :+1:

**Pull request merged**